### DISCUSSION TITLE
Queue up future dag runs from the UI

### DISCUSSION THREAD

**Discussion by will-byrne-cardano** [apache/airflow#39679] [Ideas]
Say you have a dag that needs no schedule, it is an automated one-off process that runs every so often. It would be great to have the ability to schedule a one-off run of that dag for some future time from within the UI. E.g. it is currently 2pm and you know you want to run this dag out of hours, it would be good to be able to schedule a run of the dag at 11pm for that day. 

I use Octopus Deploy in my job and they have a similar feature for deployments to the one described above. I could see this being a useful addition, hopefully others might as well.

**Comment by chaityacshah** [comment]
Could you add some more use cases on when this would be useful?
